project/funcoes.class.js

```diff
@@ -0,0 +1,29 @@
+// Classe para chamar o Json.
+function json(){
+	var qtd;
+	var retorno;
+
+	// Resgatar valores.
+	json.prototype.resgatarValores = function(){
+		$('#resultado').html('Carregando dados...');
+
+		// Estrutura de resultado.
+		$.getJSON('urls.json', function(data){
+			this.qtd = data.links.length;
+			this.retorno = '';
+			for (i = 0; i < this.qtd; i++){
+                this.retorno += '<table><tr>';
```

Comment:
It's not a good practice to concatenate DOM elements as string. You should create a object for each DOM element and manipulate them accordingly. 

There are a lot of techniques to accomplish this (and some regarding performance), but I'll keep it simple with the following example (as you are using jQuery):

```
var table = $('<table></table>');

for(var i = 10; i--;){
  let tr = $('<tr></tr>')
  	.addClass('hey')
  	.append(
  		$('<td></td>').text('just an example ' + i)
  	);  	

  table.append(tr);
}

table.appendTo('#some_container');
```